### PR TITLE
[DOCS] Add gssoc'23 timeline and pointer system

### PR DESCRIPTION
**Pull request by karthiknadar1204**
## Changes proposed
Added gssoc timeline and pointer system used for gssoc'23

### CODE REVIEW

**Review by CBID2 — APPROVED** [review]
LGTM!

**Comment by karthiknadar1204** [comment]
Any update on this pr?

**Comment by ujjawaltyagii** [comment]
It would have been easier to review if you could have provided the screen shot :)

**Comment by karthiknadar1204** [comment]
So sorry for the inconvenience, here is the ss
Screenshot 2023-07-09 at 1 54 08 PM

File: README.md [hunk 1]
```diff
@@ -128,6 +128,56 @@ You can see the live demo at: https://linkshub.vercel.app
 #### Want to add or update the descriptions of subcategories?
 Make sure to follow [CONTRIBUTING guidelines](./CONTRIBUTING.md#Adding-and-Updating-Category-Description-).
 
+
+	
+<h1>GSSOC'23 Issue TimeLine</h1>
```
Comment (rupali-codes) [inline README.md:133]:
```suggestion
## GSSOC'23 Issue TimeLine
```

File: README.md [hunk 2]
```diff
@@ -128,6 +128,56 @@ You can see the live demo at: https://linkshub.vercel.app
...
+	
+	
+- Once an issue is assigned,the assignee is expected to submit a pr for review within a week of the assignment. 
+	
+- If the assignee fails to comply with the deadline, the issue will be assigned to the next person who had who had requested to be assigned.
+	
+	
+
+<h1>GSSOC'23 Pointer System</h1>
```
Comment (rupali-codes) [inline README.md:142]:
```suggestion
## GSSOC'23 Pointer System
```

File: README.md [hunk 3]
```diff
@@ -128,6 +128,56 @@ You can see the live demo at: https://linkshub.vercel.app
...
+<h2> Level 1 - Documentation/Minor bug fix </h2>
+<h3> Points - 10 </h3>
```
Comment (rupali-codes) [inline README.md:144]:
```suggestion
#### Points - 10
```

File: README.md [hunk 4]
```diff
@@ -128,6 +128,56 @@ You can see the live demo at: https://linkshub.vercel.app
...
+- Fixing issues such as typos, brokel links, or minor performance problems.
+	
+	
+- Bug fixes are an important part of maintaining stability of and reliability of of an open-source project and every bug fix, no matter how small, contributes to the overall health of the project. 
+	
+	
+
+	
+	
+<h2> Level 2 - Enhancement of existing features </h2>
```
Comment (rupali-codes) [inline README.md:162]:
```suggestion
###  Level 2 - Enhancement of existing features
```

File: README.md [hunk 5]
```diff
@@ -128,6 +128,56 @@ You can see the live demo at: https://linkshub.vercel.app
...
+<h3> Points - 25 </h3>
```
Comment (rupali-codes) [inline README.md:163]:
```suggestion
### Points - 25 
```

File: README.md [hunk 6]
```diff
@@ -128,6 +128,56 @@ You can see the live demo at: https://linkshub.vercel.app
...
+-  Feature or enhancement contributions refer to adding new functionality to an open source project.  
+	
+- Contributors can add new features, improve existing features, or add new functionality to existing features.
+	
+
+	
+	
+	
+	
+<h2> Level 3 - Refactoring/ Adding functionalities </h2>
```
Comment (rupali-codes) [inline README.md:175]:
```suggestion
### Level 3 - Refactoring/ Adding functionalities
```

File: README.md [hunk 7]
```diff
@@ -128,6 +128,56 @@ You can see the live demo at: https://linkshub.vercel.app
...
+<h3> Points - 45 </h3>
```
Comment (rupali-codes) [inline README.md:176]:
```suggestion
#### Points - 45
```

**Review by CBID2 — APPROVED** [review]
LGTM! 😊

**Review by k-deepak04 — APPROVED** [review]

**Comment by CBID2** [comment]
For some odd reason, I can't merge this PR.

**Comment by k-deepak04** [comment]
> For some odd reason, I can't merge this PR. @rupali-codes, we need your help here.

same with me , i guess it's due to one review pending of ujjwal

**Comment by CBID2** [comment]
Why is this PR closed @karthiknadar1204?

**Comment by karthiknadar1204** [comment]
Even i dont' know, I am opening my system just now, SHould I Open another pr, can't seem to find the option to reopen the pr.
@CBID2

**Comment by karthiknadar1204** [comment]
@CBID2 @k-deepak04 @rupali-codes Please look into it in your free time.
Thank you

**Review by CBID2 — APPROVED** [review]
LGTM

**Review by k-deepak04 — APPROVED** [review]

**Comment by karthiknadar1204** [comment]
Thank you @CBID2 @k-deepak04

**Pull request merged**